src/server/scripts/Northrend/Naxxramas/boss_gothik.cpp

```diff
@@ -173,6 +173,10 @@ const Position PosSummonDead[5] =
 #define POS_X_SOUTH  2633.84f
 #define IN_LIVE_SIDE(who) (who->GetPositionY() < POS_Y_GATE)
 
+// GUID of first trigger NPC (used as offset for guid checks)
+// 0-1 are living side soul triggers, 2-3 are spectral side soul triggers, 4 is living rider spawn trigger, 5-7 are living other spawn trigger, 8-12 are skull pile triggers
+#define CGUID_TRIGGER 127618
```

Comment:
Maybe I'm alone on this, but I don't like doing anything based off arbitrary GUIDs. Not sure what sort of solution I would prefer, but GUIDs are easily malleable and hardcoding and operating off of them makes me sad.

Reply:
I agree, this will break if someone changes the guids after something like a rework for naxxramas would happen and the guids change based on creature spawns changing

Reply:
@elthehablo I've updated the CGUID approach to use coordinates instead in 7b818b2653d28618894b26f3eda9946bbacd5be3